Userland/Libraries/LibWeb/HTML/HTMLSelectElement.cpp

```diff
@@ -56,6 +58,22 @@ void HTMLSelectElement::adjust_computed_style(CSS::StyleProperties& style)
         style.set_property(CSS::PropertyID::Display, CSS::DisplayStyleValue::create(CSS::Display::from_short(CSS::Display::Short::InlineBlock)));
 }
 
+// https://html.spec.whatwg.org/multipage/form-elements.html#dom-select-size
+WebIDL::UnsignedLong HTMLSelectElement::size() const
+{
+    // The size IDL attribute must reflect the respective content attributes of the same name. The size IDL attribute has a default value of 0.
+    if (auto size_string = get_attribute(HTML::AttributeNames::size); size_string.has_value()) {
+        if (auto size = parse_integer(*size_string); size.has_value())
```

Comment:
Bit of a yakbait, but this is one type of thing where (especially since the spec says "reflect") - we could perhaps implement this in the IDL generator to prevent these types of issues in the future

Reply:
That would be a nice idea `(unsigned) long` reflects with default `0` are quite common